src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/attr/Change.java

```diff
@@ -331,6 +346,30 @@ public void setLastUpdated(Date date) {
         this.lastUpdated = date;
     }
 
+    /**
+     * Is this change a work in progress.
+     * @return change is in WIP state.
+     */
+    public boolean isWip() { return wip; }
+
+    /**
+     * Is this change a work in progress.
+     * @param boolean change is in WIP state.
+     */
+    public void setWip(boolean wip) { this.wip = wip; }
+
+    /**
+     * Is this change private.
+     * @return change is in private state.
+     */
+    public boolean isPrivate() { return isPrivate; }
```

Comment:
Hi @rsandell, @sselberg 
I try compile source gerrit trigger and I got error below:
error: maven-checkstyle-plugin:2.13:checkstyle (compile-checkstyle) @ gerrit-trigger
.......'isPrivate' hides a field.
.......Expected @param tag for 'isPrivate'

Suggest solution:
    /**
     * Is this change private.
     * @return change is in private state.
     */
    public boolean isPrivateChanged() {
        return isPrivate;
    }

    /**
     * Is this change private.
     * @param isPrivateChanged the boolean .
     */
    public void setPrivate(boolean isPrivateChanged) {
        this.isPrivate = isPrivate;
    }

Reply:
Your suggested change doesn't follow the Java bean standard. Better to fully follow it and name the field `private` but that isn't possible since it's a reserved word.

Reply:
So either change the field name to something else, like `_private` or add a checkstyle ignore directive.